owasp-top10-2016-mobile/m2/cool_games/README.md

```diff
@@ -178,14 +178,18 @@ How would you mitigate this vulnerability? After your changes, an attacker shoul
 
 ## PR solutions
 
-[Spoiler alert 🚨] To understand how this vulnerability can be mitigated, check out [these pull requests](https://github.com/globocom/secDevLabs/pulls?q=is%3Apr+is%3Aclosed+label%3AM2-OWASP-2016+label%3A%22Cool+Games%22)!
+[Spoiler alert 🚨] To understand how this vulnerability can be mitigated, check out [these pull requests][these pull requests]!
 
 ## Contributing
 
 We encourage you to contribute to SecDevLabs! Please check out the [Contributing to SecDevLabs](../../../docs/CONTRIBUTING.md) section for guidelines on how to proceed! 🎉
 
+
+[GamesIRADOScom]: https://github.com/globocom/secDevLabs/tree/master/owasp-top10-2021-apps/a9/games-irados
+[secDevLabs]: https://github.com/globocom/secDevLabs
 [API-Version-Issue]: https://github.com/globocom/secDevLabs/blob/master/docs/api-version-issue.md
 [Flutter]: https://flutter.dev/docs/get-started/install
 [VS-Code-Flutter]: https://github.com/globocom/secDevLabs/blob/master/docs/installing-flutter.md
 [Docker Install]:  https://docs.docker.com/install/
 [Docker Compose Install]: https://docs.docker.com/compose/install/
+[these pull requests]:https://github.com/globocom/secDevLabs/pulls?q=is%3Apr+is%3Aclosed+label%3AM2-OWASP-2016+label%3A%22Cool+Games%22
```

Comment:
wrong pull requests link, checkout this [PR](https://github.com/globocom/secDevLabs/pull/605/files) :wink:

as for the others readme (cool_games,  camplake-api, stegonography, panda_zap, note-box)